app/pages/reporter/facilities.tsx

```diff
@@ -18,26 +17,32 @@ class FacilitiesList extends Component<Props> {
   static isAccessProtected = true;
   static query = graphql`
     query facilitiesQuery(
-      $orderByField: String
-      $direction: String
-      $searchField: String
-      $searchValue: String
-      $organisationRowId: String
-      $offsetValue: Int
-      $maxResultsPerPage: Int
-      $reportingYear: Int
+      $operatorName: String
+      $facilityName: String
+      $applicationStatus: CiipApplicationRevisionStatus
+      $applicationIdIsNull: Boolean
+      $applicationId: Int
+      $organisationRowId: Int
+      $offset: Int
+      $max_results: Int
+      $reportingYear: Int = 2019
```

Comment:
2019 hardcoded? Could we init this to the value of openedReportingYear() || 2019 instead?

Reply:
That is to be fixed when working on https://youtrack.button.is/issue/GGIRCS-2135. 
Maybe I should hardcode it to 2020 though.
I don't think that we can do this initialization in graphql. I was thinking of creating a `default_searched_reporting_year()` function (either the currently opened one or the last one that was opened), and creating another `facility_application_by_reporting_year()` function with no parameter, which would use `default_searched_reporting_year()`. I want it to be another function so that they can remain `sql` instead of `plpgsql` (so that the query planner can optimize things)